tests/test_image_transforms.py

```diff
@@ -278,7 +278,7 @@ def test_resize(self):
         self.assertEqual(resized_image.shape, (4, 30, 40))
 
     def test_normalize(self):
-        image = np.random.randint(0, 256, (224, 224, 3)) / 255
+        image = np.random.randint(0, 256, (224, 224, 3)).astype(np.float32) / 255
```

Comment:
Why this place need to cast to fp32?
(we have a test case below for it IIUC.)
If we cast here, we no longer test with python float numbers, right?

Reply:
oh, I didn't realize this ...